### PR TITLE
RP - remove xplat/include versions

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-2.2&branch=pr-en-us-9773)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra 	looks like you can't test redirection

**Comment by tdykstra** [comment]
@Rick-Anderson yes you can -- redirection works in staging.  If you're seeing it not work, there's likely something wrong with the redirection json.

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1,574 +1,630 @@
...
-    {
-        "source_path": "aspnetcore/web-api/advanced/index.md",
-        "redirect_url": "/aspnet/core/web-api/",
-        "redirect_document_id": false
-    }
-  ] 
-}
+    "redirections": [ 
+      {   
+          "source_path": "tutorials/razor-pages-mac/index.md",
```
Comment (tdykstra) [inline .openpublishing.redirection.json]: your source path is missing aspnetcore

Comment (Rick-Anderson) [reply .openpublishing.redirection.json]: Darn. I think VSC did that.

**Pull request merged**